modules/example/src/main/scala/Minimal2.scala

```diff
@@ -53,7 +53,7 @@ object Minimal2 extends IOApp {
       }
     }
 
-  def runF[F[_]: Async: Trace: Parallel: Console]: F[ExitCode] =
+  def runF[F[_]: Async: Trace: Parallel: Console: Network]: F[ExitCode] =
```

Comment:
Instead of `parallel` this probably just needs `import cats.effect.implicits._`

Reply:
https://github.com/tpolecat/skunk/commit/425335900633ca6769293394cfb917c8a0997359 does all of the relevant tidying :)

Reply:
Looks good, can you PR that?